spec/controllers/v2/tailorings_controller_spec.rb

```diff
@@ -439,4 +439,125 @@
       end
     end
   end
+
+  context '/policies/:id/tailorings/:id/compare' do
+    describe 'GET compare' do
+      let(:canonical_profile) do
+        FactoryBot.create(
+          :v2_profile,
+          security_guide: FactoryBot.create(:v2_security_guide, rule_count: 5, os_major_version: 9),
+          ref_id_suffix: 'foo',
+          supports_minors: [6, 7, 8],
```

Comment:
Test coverage for `target_os_minor_version` outside of scope of `supports_minors` is missing